### PR TITLE
Made reME-centric climb controls

### PR DESCRIPTION
**Pull request by SumedhP**

### CODE REVIEW

**Review by bobtseattle — APPROVED** [review]
Approved other than the white space spotless issue (rightly called out).

File: src/main/java/frc/team2412/robot/Subsystems.java [hunk 1]
```diff
@@ -15,9 +15,9 @@
         public static final boolean DRIVE_ENABLED = true;
         public static final boolean DRIVER_VIS_ENABLED = true;
         public static final boolean SHOOTER_VISION_ENABLED = true;
-        public static final boolean INDEX_ENABLED = true;
-        public static final boolean INTAKE_ENABLED = true;
-        public static final boolean SHOOTER_ENABLED = true;
+        public static final boolean INDEX_ENABLED = false;
+        public static final boolean INTAKE_ENABLED = false;
+        public static final boolean SHOOTER_ENABLED = false;
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Subsystems.java:20]: Need to re-enable the subsystems...

File: src/main/java/frc/team2412/robot/commands/intake/IntakeCommand.java [hunk 2]
```diff
@@ -21,9 +21,10 @@ public void execute() {
             intakeSubsystem.intakeStop();
             return;
         }
-
+        
+        System.out.println(indexSubsystem.hasCargo() && intakeSubsystem.hasCargo());
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/commands/intake/IntakeCommand.java:25]: Please remove diagnostic print

File: src/main/java/frc/team2412/robot/Controls.java [hunk 3]
```diff
@@ -96,8 +103,12 @@ public Controls(Subsystems s) {
     }
 
     public void bindClimbControls() {
-        climbFixedArmDown.whenPressed(new RetractArmFullyCommand(subsystems.climbSubsystem));
-        climbFixedArmUp.whenPressed(new ExtendArmCommand(subsystems.climbSubsystem));
+        climbArmDown.whenPressed(new RetractArmFullyCommand(subsystems.climbSubsystem));
+        climbArmUp.whenPressed(new ExtendArmCommand(subsystems.climbSubsystem));
+
+        climbArmDownManual.whileHeld(new ClimbSetArmCommand(subsystems.climbSubsystem, -0.4));
+        climbArmUpManual.whileHeld(new ClimbSetArmCommand(subsystems.climbSubsystem, 0.4));
```
Comment (jamesdooley4) [inline src/main/java/frc/team2412/robot/Controls.java:110]: These tell the climb arm to go to position -0.4 or position 0.4, which seems incorrect. ClimbSetArmCommand currently sets an absolute position. I'm guessing there's missing code in ClimbSetArmCommand and ClimbSubsystem to enable this to instead simply set the motor power (or speed).

**Review by jamesdooley4 — CHANGES_REQUESTED** [review]

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**